packages/config/src/sitemap-plugin.js

```diff
@@ -0,0 +1,25 @@
+const { resolve } = require('path');
+const Sitemap = require('react-router-sitemap');
+const fs = require('fs');
+
+class SitemapPlugin {
+    constructor(rootFolder) {
+        this.rootFolder = rootFolder;
+    }
+
+    apply(compiler) {
+        compiler.hooks.emit.tapAsync('SitemapPlugin', (compilation, callback) => {
+            const packageJson = require(resolve(this.rootFolder, 'package.json'));
+
+            const paths = Object.values(packageJson.routes).map(value => value.path || value);
```

Comment:
One final adjustment so the build won't fail when `routes` are not defined
```suggestion
            const paths = Object.values(packageJson?.routes || {}).map(value => value.path || value);
```